ghost/core/core/server/services/email-service/email-provider-factory.js

```
@@ -0,0 +1,63 @@
+        };
+
+        const mailgunClient = new MailgunClient({config, settings});
+
+        return new MailgunEmailProvider({
+            mailgunClient,
+            errorHandler
+        });
+    }
+
+    // This should never be reached due to validation, but keeping for safety
+    throw new Error(`Provider '${provider}' is not implemented`);
```

ESLint

[error] Direct calls to new Error() are not allowed. Please use @tryghost/errors.

(ghost/ghost-custom/no-native-error)